### ISSUE TITLE
Minimap hides after peek definition or find all references

### ISSUE THREAD

**Issue by jisbert** [Microsoft/vscode#29263]
- VSCode Version: Code 1.13.1 (379d2efb5539b09112c793d3d9a413017d736f89, 2017-06-14T18:21:47.485Z)
- OS Version: Windows_NT ia32 10.0.14393
- Extensions: 

|Extension|Author|Version|
|---|---|---|
|mssql|ms-mssql|1.0.0|
|csharp|ms-vscode|1.10.0|
|PowerShell|ms-vscode|1.4.0|
|java|redhat|0.6.0|;

---

Steps to Reproduce:

1. Enable minimap by placing this in user or workspace settings or both:
```json
{
    "editor.minimap.enabled": true,
    "editor.minimap.renderCharacters": false,
    "editor.minimap.maxColumn": 120
}
```
2. Peek definition and close by clicking on the x button (minimap hides) or by pressing <kbd>Esc</kbd> and then click on the editor, minimap hides then

This gif illustrates this behaviour:

![minimap bug](https://user-images.githubusercontent.com/410091/27432920-9d4b863a-5752-11e7-82e1-51a520831bf6.gif)

**Comment by jens1o** [comment]
This is fixed in latest insider, also a duplicate.

https://github.com/Microsoft/vscode/issues/28093

**Comment by jisbert** [comment]
@jens1o Yes, you are right, I close this issue. While following the [contributing issues guide](https://github.com/Microsoft/vscode/blob/master/CONTRIBUTING.md) I searched related issues but I didn't find any. Sorry for the inconvenience and thank you.
